rpmio/macro.c

```diff
@@ -45,6 +45,12 @@ enum macroFlags_e {
     ME_USED	= (1 << 1),
 };
 
+enum checkConditionType {
+    CHK_NO		= 0,
+    CHK_BASIC		= (1 << 0),
+    CHK_TRIPLE		= (1 << 1),
+};
```

Comment:
CHK_NO sounds like something that wants to pair with CHK_YES, but this is something entirely different. 

I didn't really look whether it makes sense in the rest of the code, I'd expect the to be a CHK_EXISTS bit, and if its of the triple format, there'd be an extra bit set on that, instead of being entirely separate entities. But please rethink these names anyhow.